lib/dom.js

```diff
@@ -751,6 +752,20 @@ declare class HTMLFormElement extends HTMLElement {
     submit(): void;
 }
 
+declare class HTMLIFrameElement extends HTMLElement {
+    allowFullScreen: boolean;
+    frameBorder: number;
+    height: number;
+    marginHeight: number;
+    marginWidth: number;
+    name: string;
+    scrolling: string;
+    sandbox: string;
```

Comment:
Wait a minute, `DOMTokenList` already exists: https://github.com/facebook/flow/blob/master/lib/dom.js#L400

I'll use that :)